easybuild/easyconfigs/c/cpp-netlib/cpp-netlib-0.13.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'cpp-netlib'
+version = '0.13.0'
+
+homepage = 'https://cpp-netlib.org'
+description = """A collection of open-source libraries for high level network programming."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+sources = [{
+    'filename': '%(name)s-%(version)s-git.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/cpp-netlib',
+        'repo_name': 'cpp-netlib',
+        'commit': 'cpp-netlib-0.13.0-final',
+        'recursive': True,
+        'keep_git_dir': True,
```

Comment:
I've tried and failed to use a mixture of other dependencies (including `cxxopts`, and `googletest`) but couldn't manage to get everything in the right place to build correctly.  (CMake often can't find the `CMakeLists.txt` files).  Since the git submodules are uniquely specified by hashes (see https://github.com/cpp-netlib/cpp-netlib/tree/cpp-netlib-0.13.0-final/deps) maybe this is sufficient to ensure reproducible builds across different systems?